prereise/cli/data_sources.py

```diff
@@ -0,0 +1,130 @@
+import logging
+from abc import ABCMeta, abstractmethod
+
+from powersimdata.input.grid import Grid
+
+from prereise.cli.helpers import validate_date, validate_file_path
+from prereise.gather.winddata.rap import rap
+
+
+class DataSource(metaclass=ABCMeta):
+    """Abstract class that defines a strict interface
+        that future data sources need to follow in order
+        to seamlessly integrate with the parser.
```

Comment:
The two lines above should not be indented

Reply:
updated! thanks